src/OpenTelemetryEngine/Logging/FunctionsLogger.cs

```diff
@@ -0,0 +1,40 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.Extensions.Logging;
+using OpenTelemetryEngine.ResponseObjects;
+
+namespace OpenTelemetryEngine.Types
+{
+    public class FunctionsLogger
+    {
+        public ILogger logger;
+
+        public FunctionsLogger(ILogger logger) {  this.logger = logger; }
+
+        public void Log(object? logItem, string? level, Exception? exception = null) 
+        {
+            if (Enum.TryParse(typeof(LogLevel), level, out var _logLevel)) {
```

Comment:
What if `Enum.TryParse` returns false? I think this deserves an ArgumentException.

Reply:
Done